app/views/posts/index.html.erb

```diff
@@ -1,12 +1,22 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1>list of the posts shown for a designated user.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div id="user-container">
+   <%= render partial: "users/user_info", locals: { user: @user } %>
+</div>
+
+<ul id="post-container">
+  <% @posts.each_with_index do |post, i| %>
+    <li>
+      <%= link_to "/users/#{@user.id}/posts/#{post.id}", id: "post" do %>
+        <%= render partial: "posts/post_info", locals: { post: post } %>
+      <% end %>
+      <ul id="comment-container">
+        <% post.five_most_recent_comments.each do |comment| %>
+          <li id="comment">
+            <p><%= comment.user.name %>: <%= comment.text %></p>
+          </li>
+        <% end %>
+      </ul>
+    </li>
+  <% end %>
+</ul>
+
+<%= will_paginate %>
```

Comment:
- [ ] when I navigate to post index I get an error of unknown method paginate, kindly fix this
![image](https://github.com/ethred/blog_app/assets/66097371/49e6ba3d-1b50-4347-bd5d-1b83fa830aab)